src/core/utils/calculateProfit.ts

```diff
@@ -1,23 +1,28 @@
+import { BaseToken } from '../../types/adapter'
+
 export function calculateProfit(
-  deposits: Record<string, number | undefined> | undefined,
-  withdrawals: Record<string, number | undefined> | undefined,
-  currentValues: Record<string, number | undefined> | undefined,
-  previousVales: Record<string, number | undefined> | undefined,
-): Record<string, number> {
+  deposits: Record<string, bigint>,
```

Comment:
can you right click on calculateProfit(...) then click refactor, then convert this to accept an object please (I know it was me that implemented but its confusing)

Reply:
Done.